test/actions/AuthorisedActionSpec.scala

```diff
@@ -313,27 +307,9 @@ class AuthorisedActionSpec extends ControllerUnitTest
         }
       }
 
-      "[EMA disabled] results in an AuthorisationException error being returned from Auth" should {
-        "return a redirect to the agent error page" in new AgentTest {
-          mockMultipleAgentsSwitch(false)
-
-          object AuthException extends AuthorisationException("Some reason")
-          mockAuthReturnException(AuthException, primaryAgentPredicate(mtdItId))
-
-          val result: Future[Result] = testAuth.agentAuthentication(testBlock)(
-            request = FakeRequest().withSession(fakeRequestWithMtditidAndNino.session.data.toSeq :_*),
-            hc = emptyHeaderCarrier
-          )
-
-          status(result) shouldBe SEE_OTHER
-          redirectUrl(result) shouldBe s"$baseUrl/error/you-need-client-authorisation"
-        }
-      }
-
-      "[EMA enabled] results in an AuthorisationException error being returned from Auth" should {
+      "results in an AuthorisationException error being returned from Auth" should {
 
         "render an ISE page when secondary agent auth call also fails with non-Auth exception" in new AgentTest {
-          mockMultipleAgentsSwitch(true)
 
           mockAuthReturnException(InsufficientEnrolments(), primaryAgentPredicate(mtdItId))
           mockAuthReturnException(new Exception("bang"), secondaryAgentPredicate(mtdItId))
```

Comment:
Could we possibly add a similar test for the following scenario:

```
mockAuthReturnException(InsufficientEnrolments(), primaryAgentPredicate(mtdItId))
mockAuthReturnException(InsufficientEnrolments(), secondaryAgentPredicate(mtdItId))
```

Maybe the test could be called
"render an ISE page when secondary agent auth call also fails with insufficient enrolments"